README.md

```diff
@@ -1,14 +1,20 @@
 # Twenty Twenty-One
 
-Contributors: wordpressdotorg  
-Requires at least: 5.3  
-Tested up to: 5.5  
-Requires PHP: 5.6  
-License: GPLv2 or later  
-License URI: http://www.gnu.org/licenses/gpl-2.0.html  
+Contributors: wordpressdotorg
+Requires at least: 5.3
+Tested up to: 5.5
+Requires PHP: 5.6
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
 ## Description
 
+Twenty Twenty-One is a blank canvas for your ideas - it makes the block editor your best brush.
```

Comment:
This should be an em dash —

Reply:
Actually, let's drop the em dash entirely and go with:

> Twenty Twenty-One is a blank canvas for your ideas and it makes the block editor your best brush.

Reply:
Made this change